doc/sphinx/proof-engine/tactics.rst

```diff
@@ -71,22 +74,22 @@ specified, the default selector is used.
 Bindings list
 ~~~~~~~~~~~~~~~~~~~
 
-Tactics that take a term as argument may also support a bindings list,
+Tactics that take a term as an argument may also support a bindings list,
 so as to instantiate some parameters of the term by name or position.
 The general form of a term equipped with a bindings list is ``term with
 bindings_list`` where ``bindings_list`` may be of two different forms:
 
 .. _bindings_list_grammar:
 
   .. productionlist:: `bindings_list`
-     bindings_list : (ref := `term`) ... (ref := `term`)
+     bindings_list : (`ref` := `term`) ... (`ref` := `term`)
                    : `term` ... `term`
 
-+ In a bindings list of the form :n:`{* (ref:= term)}`, :n:`ref` is either an
++ In a bindings list of the form :n:`{+ (@ref:= @term)}`, :n:`@ref` is either an
   :n:`@ident` or a :n:`@num`. The references are determined according to the type of
-  ``term``. If :n:`ref` is an identifier, this identifier has to be bound in the
+  ``term``. If :n:`@ref` is an identifier, this identifier has to be bound in the
```

Comment:
```suggestion
   :n:`@term`. If :n:`@ref` is an identifier, this identifier has to be bound in the
```

Same change to do three other times in this paragraph, and one missing `@` in the next paragraph.